src/Storages/StorageMongoDB.h

```diff
@@ -59,6 +59,7 @@ class StorageMongoDB final : public IStorage
 
 private:
     void connectIfNotConnected();
+    ColumnsDescription getTableStructureFromData();
```

Comment:
I think we should also use it in `TableFunctionMongoDB::getActualTableStructure` so schema inference will work for query like `DESC mongodb(...)`